src/Common/Commands.Common.Compute.Tests/Commands.Common.Compute.Tests.csproj

```diff
@@ -0,0 +1,108 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should go in the profile tests project - if it does not, you will need to create a lot of test infrastructure to make this execute like the other tests.